### PR TITLE
Add query parameter on coverage badge image url

### PR DESCRIPTION
**Pull request by x-hgg-x**

### CODE REVIEW

**Comment by x-hgg-x** [comment]
This should be tested by manually updating the badge on S3.

**Comment by lopopolo** [comment]
I'm still missing cache control headers from cloudfront. I think I need to deploy a cloudfront function to do this.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # strftime-ruby
 
 [![GitHub Actions](https://github.com/artichoke/strftime-ruby/workflows/CI/badge.svg)](https://github.com/artichoke/strftime-ruby/actions)
-[![Code Coverage](https://codecov.artichokeruby.org/strftime-ruby/badges/flat.svg)](https://codecov.artichokeruby.org/strftime-ruby/index.html)
+[![Code Coverage](https://codecov.artichokeruby.org/strftime-ruby/badges/flat.svg?nocache=1)](https://codecov.artichokeruby.org/strftime-ruby/index.html)
```
Comment (lopopolo) [inline README.md]: bust cache again

```suggestion
[![Code Coverage](https://codecov.artichokeruby.org/strftime-ruby/badges/flat.svg?nocache=2)](https://codecov.artichokeruby.org/strftime-ruby/index.html)
```

**Review by lopopolo — APPROVED** [review]

**Comment by lopopolo** [comment]
infra changes were merged in https://github.com/artichoke/project-infrastructure/pull/336. The badge in the readme re-rendered.

**Pull request merged**